packages/iris-grid/src/sidebar/TableCsvExporter.tsx

```diff
@@ -388,10 +388,16 @@ class TableCsvExporter extends Component<
                   : null}
               </span>
             </Radio>
-            <Radio
-              value={TableCsvExporter.DOWNLOAD_ROW_OPTIONS.CUSTOMIZED_ROWS}
-              data-testid="radio-csv-exporter-customized-rows"
-            >
+            <div className="d-flex">
+              <Radio
+                value={TableCsvExporter.DOWNLOAD_ROW_OPTIONS.CUSTOMIZED_ROWS}
+                data-testid="radio-csv-exporter-customized-rows"
+                UNSAFE_className="mr-2 pr-1"
+                // Empty aria-label workaround is needed to satisfy accessibility requirement since the
+                // Radio's label is rendered separately. This prevents accessibility errors that
+                // can trigger race conditions with Service Worker updates.
+                aria-label=" "
```

Comment:
Should not have a blank aria-label
```suggestion
                aria-label="First or last number of rows"
```